.babelrc

```diff
@@ -16,14 +16,16 @@
       "displayName": true
     }],
     "inline-react-svg",
+    "lodash",
+    "loadable-components/babel",
     ["module-resolver", {
       "extensions": [".js", ".jsx", ".ts", ".tsx"],
       "root": ["./src"],
       "alias": {
         "reaction": "@artsy/reaction/dist"
       }
     }],
-    "relay"
+    ["relay", { "artifactDirectory": "./src/__generated__", "extensions": ".ts" }]
```

Comment:
Does this `extensions` key here actually do anything?

Reply:
I've removed this